polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -1440,9 +1440,9 @@ ValueCategory MLIRScanner::CommonFieldLookup(clang::QualType CT,
             })
             .Case<sycl::AccessorType, sycl::AccessorImplDeviceType,
                   sycl::AccessorSubscriptType, sycl::AtomicType,
-                  sycl::GetScalarOpType, sycl::GroupType, sycl::ItemBaseType,
-                  sycl::ItemType, sycl::NdItemType, sycl::NdRangeType,
-                  sycl::VecType>([&](auto ElemTy) {
+                  sycl::MultiPtrType, sycl::GetScalarOpType, sycl::GroupType,
```

Comment:
Can we place the types in alphabetical order pls.

Reply:
Done